content/learning-how-to-teach-coders/quizzes/_index.md

```diff
@@ -1,5 +1,45 @@
 ---
-_db_id: 649
 content_type: topic
 title: Quizzes
----
\ No newline at end of file
+---
+
+## What The Sessions Should Be About
```

Comment:
The following heading is much easier on the eye. It's generally good to just use sentence case unless there is a word that needs extra emphasis

```
## What the session should be about
```